src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -121,9 +125,14 @@ private Task DoInitialFetch()
             int retryCount = 0;
             while (!this.sonarQubeService.IsConnected && retryCount < 30)
             {
+                if (this.initialFetchCancellationTokenSource.IsCancellationRequested)
+                {
+                    return Task.CompletedTask;
+                }
```

Comment:
We can get into this wait loop in two cases:
1) the connection is in progress but not yet complete, and Roslyn starts doing an analysis (i.e. the original bug)
2) if we open a bound solution but the SonarQube server is not available. In this case, the loop will time out after thirty seconds and the issues will not be suppressed.